.idea/runConfigurations/Ensemble8_jar.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Do we really need these run configurations (except `Run_Robot_Tests.xml`, which is useful)

Reply:
Yeah, I agree. Removed those and left the robot tests.